### PR TITLE
Add Beekeeper Studio v1.1.3

### PR DESCRIPTION
**Pull request by kochalex**
Adds Beekeeper Studio.app, an open source SQL client.

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/beekeeper-studio.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'beekeeper-studio' do
+  version '1.1.2'
+  sha256 '5850f254c5506ec65d39737200a1409c1e670c2a1644e7b1d72deaacf4d90f06'
+
+  # github.com/beekeeper-studio/beekeeper-studio was verified as official when first introduced to the cask
+  url 'https://github.com/beekeeper-studio/beekeeper-studio/releases/download/v1.1.2/Beekeeper-Studio-1.1.2.dmg'
```
Comment (vladimyr) [inline Casks/beekeeper-studio.rb:6]:
```suggestion
  url "https://github.com/beekeeper-studio/beekeeper-studio/releases/download/v#{version}/Beekeeper-Studio-#{version}.dmg"
```

File: Casks/beekeeper-studio.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+cask 'beekeeper-studio' do
+  version '1.1.2'
+  sha256 '5850f254c5506ec65d39737200a1409c1e670c2a1644e7b1d72deaacf4d90f06'
+
+  # github.com/beekeeper-studio/beekeeper-studio was verified as official when first introduced to the cask
```
Comment (vladimyr) [inline Casks/beekeeper-studio.rb:5]: Trailing slash is actually significant here:
```suggestion
  # github.com/beekeeper-studio/beekeeper-studio/ was verified as official when first introduced to the cask
```

Comment (kochalex) [reply Casks/beekeeper-studio.rb:5]: Thanks for the reviews & suggestions

**Comment by vitorgalvao** [comment]
Thank you, but we’ll be rejecting this on account of [not being notable enough](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):

> + The Cask is too obscure. Examples:
>   + A self-submitted app from a GitHub repository that is [not notable enough](https://github.com/Homebrew/homebrew-cask/pull/28103) (under 30 forks, 30 watchers, and 75 stars).
>   + [Electronic Identification (eID) software](https://github.com/Homebrew/homebrew-cask/issues/59021).

**Comment by liamdawson** [comment]
So is it expected to meet the forks, watchers AND stars count? It has 114 stars now, but only 4 forks/5 watchers.

**Comment by vladimyr** [comment]
You got it right https://github.com/SMillerDev/brew/blob/42116f6/Library/Homebrew/utils/notability.rb#L18-L21

**Comment by liamdawson** [comment]
```
if (metadata["forks_count"] < 30) && (metadata["subscribers_count"] < 30) && (metadata["stargazers_count"] < 75)
```

To me, that says it's not noteworthy enough if ALL of those conditions aren't met, whereas the current stargazers count is more than 75.

**Comment by vladimyr** [comment]
Yeah, you are right. At the time of PR creation, the star count was below the threshold.

**Comment by taylorthurlow** [comment]
Maybe this can be re-opened, it's well beyond the notability threshold now.

**Comment by vladimyr** [comment]
> Maybe this can be re-opened, it's well beyond the notability threshold now.

Version got bumped in the meantime so it is probably for the best to create a new one.

**Pull request closed without merging**